### PR TITLE
CHANGES: prepare for v1.2 release

### PR DESCRIPTION
**Pull request by ejoerns**
This shortly describes changes done since v1.1 release

Signed-off-by: Enrico Jorns <ejo@pengutronix.de>

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -1,6 +1,35 @@
...
+
+* Let rauc-hawkbit-updater use the recent InstallBundle() DBus method instead of
+  legacy Install() method. [#129]
+
+.. rubric:: Bug Fixes
+
+* fixed NULL pointer dereference if build_api_url() is called for base
+  deployment URL with having GLIB_USING_SYSTEM_PRINTF defined [#115]
+* fixed compilation against musl by not including glibc-specific
+  bits/types/struct_tm.h [#123] (by Zygmunt KrynickiA)
```
Comment (Bastian-Krause) [inline CHANGES:14]:
```suggestion
  bits/types/struct_tm.h [#123] (by Zygmunt Krynicki)
```

File: CHANGES [hunk 2]
```diff
@@ -1,6 +1,35 @@
...
+
+.. rubric:: Code
+
+* drop some unused variable [#126]
```
Comment (Bastian-Krause) [inline CHANGES:18]:
```suggestion
* drop some unused variables [#126]
```

File: CHANGES [hunk 3]
```diff
@@ -1,6 +1,35 @@
...
+
+.. rubric:: Testing
+
+* enable and fix testing for ipv6 addresses [#116]
+* enhance test output by not aborting to early on process termination [#128]
```
Comment (Bastian-Krause) [inline CHANGES:23]:
```suggestion
* enhance test output by not aborting too early on process termination [#128]
```

File: CHANGES [hunk 4]
```diff
@@ -1,6 +1,35 @@
...
+* set proper names for python logger [#127]
+
+.. rubric:: Documentation
+
+* corrected retry_wait default value in reference [#118]
+* suggest using systemd-tmpfiles for creating and managing tmp directories
+  [#124] (by Jonas Licht)
```
Comment (Bastian-Krause) [inline CHANGES:30]: We should probably mention that this is only relevant for plain RAUC bundles.

File: CHANGES [hunk 5]
```diff
@@ -1,6 +1,35 @@
...
+* fixed compilation against musl by not including glibc-specific
+  bits/types/struct_tm.h [#123] (by Zygmunt KrynickiA)
+
+.. rubric:: Code
+
+* drop some unused variable [#126]
+
+.. rubric:: Testing
+
+* enable and fix testing for ipv6 addresses [#116]
```
Comment (Bastian-Krause) [inline CHANGES:22]:
```suggestion
* enable and fix testing for IPv6 addresses [#116]
```

**Review by Bastian-Krause — CHANGES_REQUESTED** [review]
Thanks for preparing the release.

File: CHANGES (hunk 1 above)
Comment (ejoerns) [reply CHANGES]: oups, this was vim, not me :smirk:

File: CHANGES (hunk 2 above)
Comment (ejoerns) [reply CHANGES]: fixed

File: CHANGES (hunk 3 above)
Comment (ejoerns) [reply CHANGES]: fixes

File: CHANGES (hunk 4 above)
Comment (ejoerns) [reply CHANGES]: I've adapted the line

File: CHANGES (hunk 5 above)
Comment (ejoerns) [reply CHANGES]: fixed

File: CHANGES [hunk 6]
```diff
@@ -1,6 +1,35 @@
...
 
+.. rubric:: Enhancements
+
+* Let rauc-hawkbit-updater use the recent InstallBundle() DBus method instead of
+  legacy Install() method. [#129]
+
+.. rubric:: Bug Fixes
+
+* fixed NULL pointer dereference if build_api_url() is called for base
+  deployment URL with having GLIB_USING_SYSTEM_PRINTF defined [#115]
```
Comment (Bastian-Krause) [inline CHANGES:12]:
```suggestion
  deployment URL without having GLIB_USING_SYSTEM_PRINTF defined [#115]
```

**Review by Bastian-Krause — CHANGES_REQUESTED** [review]
I found one more mistake, see below.

I don't know whether we should start each sentence with a capital letter, though. This wasn't consistent from the beginning. But at the end of the day, it's not important.

**Pull request merged**